config/default.yml

```diff
@@ -2643,6 +2643,15 @@ Style/ExpandPathArguments:
   Enabled: true
   VersionAdded: '0.53'
 
+Style/FloatDivision:
+  Description: 'Prefer either use fdiv or convert the left-side coersion for float division.'
+  Enabled: true
+  VersionAdded: '0.72'
+  EnforcedStyle: left_coerce
+  SupportedStyles:
+    - left_coerce
+    - fdiv
```

Comment:
I think it would be good to add `Reference: https://github.com/rubocop-hq/ruby-style-guide/issues/628` to the reference. It will help to understand how the rules were added.

Reply:
Fixed.